crates/l2/contracts/deployer.rs

```diff
@@ -16,6 +16,24 @@ use std::{
 };
 use tracing::warn;
 
+type SetupResult = (
+    Address,
+    SecretKey,
+    Address,
+    Address,
+    Address,
+    EthClient,
+    PathBuf,
+);
```

Comment:
Since we are doing this, we may use a struct to have the output organized.

Reply:
Done in [6c4a69f](https://github.com/lambdaclass/ethrex/pull/1290/commits/6c4a69f429093559e31eb9dec49958bcdd45ea2d)